io/eolearn/io/local_io.py

```diff
@@ -39,18 +42,39 @@ def execute(self, eopatch, *, filename):
         feature_type, feature_name = next(self.feature(eopatch))
         array = eopatch[feature_type][feature_name]
 
-        if self.band_count == 1:
-            array = array[..., 0]
-
         dst_shape = array.shape
-        dst_transform = rasterio.transform.from_bounds(*eopatch.bbox, width=dst_shape[1], height=dst_shape[0])
+        filename_list = []
+        print(self.date_count)
```

Comment:
Please remove prints from the code. Instead we use `logging` (on debug level) throughout `eo-learn` package. I think it might be a good idea to use it somewhere in this task.